Index: gawk-5.2.2/Makefile.in
===================================================================
--- gawk-5.2.2.orig/Makefile.in
+++ gawk-5.2.2/Makefile.in
@@ -280,7 +280,7 @@ am__define_uniq_tagged_files = \
   unique=`for i in $$list; do \
     if test -f "$$i"; then echo $$i; else echo $(srcdir)/$$i; fi; \
   done | $(am__uniquify_input)`
-DIST_SUBDIRS = support . extension extras doc awklib po test
+DIST_SUBDIRS = support . extension  doc awklib po test
 am__DIST_COMMON = $(srcdir)/Makefile.in $(srcdir)/configh.in \
 	$(srcdir)/gawkbug.in $(top_srcdir)/build-aux/ar-lib \
 	$(top_srcdir)/build-aux/compile \
@@ -514,7 +514,7 @@ EXTRA_DIST = \
 
 # Build in awklib after in doc, since we want to extract
 # sample files if doc/gawk.texi changed.
-SUBDIRS = support . $(am__append_1) extras doc awklib po test
+SUBDIRS = support . $(am__append_1)  doc awklib po test
 bin_SCRIPTS = gawkbug
 include_HEADERS = gawkapi.h
 BUILT_SOURCES = $(srcdir)/pc/Makefile.tst
